src/content/docs/new-relic-control/agent-control/setup.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: "Set up Agent Control"
+metaDescription: "Learn how to install and configure Agent Control in a Kubernetes cluster."
+freshnessValidatedDate: never
+---
+
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+There are two ways to install Agent Control:
+
+## Guided
+Log in to New Relic. In Integrations & Agents, click **Install Agent Control**, and then follow the on-screen instructions. 
+
+## Manual
+Follow the steps below to manually install Agent Control:
+
+<Steps>
+<Step>
+  ### Add the New Relic Helm repository [#add-repo]
+
+    1. Add the New Relic Helm repository to your local Helm environment by running:
+      ```shell
+            helm repo add newrelic https://helm-charts.newrelic.com
+      ```
+
+    2. If the New Relic Helm repo is already present in your local Helm environment, synchronize your environment with the upstream repos:
+      ```shell
+          helm repo update
+      ```
+</Step>
+
+<Step>
+  ### Define and validate configuration [#validate]
+
+    1. Create [a YAML file](https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml) with your configuration settings.
+
+    2. Ensure that your configuration file is correctly set up with the necessary parameters, such as your cluster name, license key, and desired managed agents.
+
```

Comment:
I think that the guided install is -sadly- almost a must now, how could the customer retrieve the client_secret otherwise? @davidgit

Reply:
I think it would be more relevant in the future, right now it risks to be too complex for users @davidgit

Reply:
I agree that the Guided Install is the only way right now (I don't think we have docs for System Identity API)